packages/block-editor/src/components/block-mobile-toolbar/index.native.js

```diff
@@ -6,50 +6,60 @@ import { Keyboard, View } from 'react-native';
 /**
  * WordPress dependencies
  */
-import { ToolbarButton } from '@wordpress/components';
-import { __, sprintf } from '@wordpress/i18n';
 import { withDispatch, withSelect } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
-import { trash } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 import BlockMover from '../block-mover';
+import BlockActionsMenu from './block-actions-menu';
 import { BlockSettingsButton } from '../block-settings';
 
+const BREAKPOINTS = {
```

Comment:
Could you please add a comment about why we need that?

Reply:
Will add it.